### PR TITLE
GIX-1686: NnsNeuronAdvancedSection Part 1

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

We are shuffling the data of the neuron detail page.

In this PR: First step of the Advanced Section in NNS neuron detail page.

# Changes

* Implement the `NnsNeuronAdvancedSection`. PENDING actions.
* Add `neuron` prop to `NnsNeuronAdvancedSection` in `NnsNeuronDetail`.
* Fix `Hash` style issue with copy button.

# Tests

* Add test file for component `NnsNeuronAdvancedSection` with PO.

# Todos

Not worth an entry yet.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2023-07-18 at 09 56 07

**Comment by lmuntaner** [comment]
@mstrasinskis @peterpeterparker please review

File: frontend/src/lib/components/ui/Hash.svelte [hunk 1]
```diff
@@ -38,6 +38,7 @@
     &.flowLessCopy {
       // add padding to the right of the text to make space for the copy icon (button + padding)
       padding-right: calc(var(--padding-4x) + var(--padding-0_5x));
+      position: relative;
```
Comment (mstrasinskis) [inline frontend/src/lib/components/ui/Hash.svelte:41]: 👍

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

**Pull request merged**